MapboxNavigation/InstructionsCardViewController.swift

```diff
@@ -121,6 +121,10 @@ open class InstructionsCardViewController: UIViewController {
     
     @objc func orientationDidChange(_ notification: Notification) {
         instructionsCardLayout.invalidateLayout()
+        
+        instructionsCardLayout.collectionView?.isPagingEnabled = false
```

Comment:
Since https://github.com/mapbox/mapbox-navigation-ios/pull/2755 has landed to `main` rebasing will be needed to resolve outstanding conflicts. This part is no longer needed I guess.

Reply:
The point of this PR is to fix swiping for right-to-left languages when using instructions cards. I haven't pushed a commit of the fix yet, but it would remove most of the added code from this commit history (since I separated into the PR I merged this morning). I can either close this PR and do a separate one, or just push the changes on this branch that I currently have. I think that the conflict will not exist after I do that.